modules/image_widget_crop/intro.md

```diff
@@ -1,30 +1,60 @@
 # ImageWidgetCrop module
 
-[ImageWidgetCrop](https://www.drupal.org/sandbox/woprrr/2571403) provides an interface for using the features of the [Crop API](https://www.drupal.org/project/crop). This widget provide an UX for use a crop on all fields images. This module have particularity to purpose capability to crop the same image by 'Crop type' configured. It's very usefull for editorial sites or media management sites.
+[ImageWidgetCrop](https://www.drupal.org/project/image_widget_crop) provides an interface for using the features of the [Crop API](https://www.drupal.org/project/crop). This widget provide an UX for use a crop on all fields images. This module have particularity to purpose capability to crop the same image by 'Crop type' configured. It's very usefull for editorial sites or media management sites.
 
-## Try me
+## Try demo module
+You can Test ImageWidgetCrop in action directly with the sub-module "ImageWidgetCrop example" to test differents usecase of this module.
 
-You can Test ImageWidgetCrop in action directly with the sub-module, "ImageWidgetCrop example" to test different usecases of this module.
+You can try the module live with the website dedicated to demonstrate the module [DEMO SITE](http://www.image-widget-crop.com) with demo user 
+Psw: "demo"
+User: "demo"
+
+You can also see [Demo vidéo](https://www.youtube.com/watch?v=Ioo_bYnCOOg)
 
 ## Requirements
 
 1. [Crop API](https://www.drupal.org/project/crop)
+1. [Cropper Library](https://github.com/fengyuanchen/cropper)
+
+##Installation
+
+1. Download and extract the module to your (`sites/all/modules/contrib`) folder.
```

Comment:
I'd rather say "Download the module using standard install process." People might have different directory structure for example.